tests/test_forecast.py

```diff
@@ -13,7 +14,7 @@ def test_forecast():
     client = MeteoFranceClient()
 
     weather_forecast = client.get_forecast(latitude=48.8075, longitude=2.24028)
```

Comment:
When testing with Montréal, Canada:
`weather_forecast = client.get_forecast(latitude=45.5016889, longitude=-73.567256)`

for `current, got`: `KeyError: 1591556400`
for `nearest` got dt: 1591552800

my dt was 1591556947

https://www.epochconverter.com

PS: `probability_forecast` raised `KeyError: 'probability_forecast'`, as it's available only in France. I think we should handle it, returning an empty list can be ok.

Reply:
Error came from for world cities, forecast don't give a forecast every  hours.
We agreed on an returning `nearest_forecast` when current is not found.